nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/command/LatencyMonitorComputation.java

```diff
@@ -85,20 +86,34 @@ public void processTimer(ComputationContext context, String key, long timestamp)
             return;
         }
         debug(String.format("Monitor latency %d/%d", count - remaining, count));
+        List<LogPartitionGroup> toRemove = new ArrayList<>();
         for (LogPartitionGroup logGroup : logGroups) {
-            List<Latency> latencies = getLatenciesForPartition(logGroup, codec);
-            if (latencies.isEmpty()) {
+            try {
+                List<Latency> latencies = getLatenciesForPartition(logGroup, codec);
+                if (latencies.isEmpty()) {
+                    continue;
+                }
+                Latency groupLatency = Latency.of(latencies);
+                publishMetrics(groupLatency, String.format("%s%s.%s.all.", basePrefix, logGroup.group, logGroup.name));
+                for (int partition = 0; partition < latencies.size(); partition++) {
+                    publishMetrics(latencies.get(partition),
+                            String.format("%s%s.%s.p%02d.", basePrefix, logGroup.group, logGroup.name, partition));
+                }
+            } catch (IllegalArgumentException e) {
+                log.warn("Cannot read latency, skipping group:partition: " + logGroup, e);
```

Comment:
If this is an expected problem do we need the stack trace? Maybe just add a log of it only at debug level?

Reply:
I don't want to ignore silently invalid logs, the user may have provided wrong parameters,
also this is from the stream.sh command line output not in our server.log